cordformation/src/main/kotlin/net/corda/plugins/Dockerform.kt

```diff
@@ -100,7 +100,7 @@ open class Dockerform @Inject constructor(objects: ObjectFactory) : Baseform(obj
                             "$nodeBuildDir/drivers:/opt/corda/drivers"
                     ),
                     "environment" to listOf("ACCEPT_LICENSE=\${ACCEPT_LICENSE}"),
-                    "ports" to listOf(it.rpcPort.get(), it.config.getInt("sshd.port")),
+                    "ports" to listOf(it.rpcPort.get(), "\"${it.config.getInt("sshd.port")}:${it.config.getInt("sshd.port")}\""),
```

Comment:
Why does the `ports` element suddenly need to be surrounded by quotes, i.e. the `\"` characters at the beginning and end?

> @chrisr3 Would be possible to have the line for the port mapping without the single quotes?

Funny you should ask.... :wink: